app/views/doorkeeper/applications/show.html.erb

```diff
@@ -23,9 +23,9 @@
     </table>
   </div>
 
-  <div class="col-md-4">
+  <div class="medium-4">
     <h3>Actions</h3>
```

Comment:
I don't see the need for a heading that says "Actions" above these buttons that do actions. Let's remove it.